scripts/README.md

```diff
@@ -0,0 +1,64 @@
+---
+page_type: script
+languages:
+- powershell
+products:
+- azure
+- azure-digital-twins
+description: "How to deploy Azure Digital twins using automated powershell scripts"
+---
+
+# Use Automated scripts to deploy Azure Digital Twins
+Deploy.ps1 is an Azure Digital Twins code sample used to deploy an Azure Digital Twins instance and set the required permissions. It can also be used as a starting point for writing your own scripted interactions.
+
+## Prerequisites
+
+* You'll need Azure subscription. You can set one up for free [here](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* You'll need a resource group in your Azure subscription. You can create your resource group using [this](https://docs.microsoft.com/azure/azure-resource-manager/management/manage-resource-groups-cli) article.
+* Download [Azure Digital Twins samples](https://docs.microsoft.com/samples/azure-samples/digital-twins-samples/digital-twins-samples/) to your machine. In the downloaded sample folder, the deployment script is located at _Azure_Digital_Twins_samples.zip > scripts > **deploy.ps1**_.
+* To be able to complete all the steps in this article, you need to be classified as an Owner in your Azure subscription.
```

Comment:
Since this is a script and not an article, we should say something like "To be able to run the full script, you need..."